### DISCUSSION TITLE
Accessing document providers from termux

### DISCUSSION THREAD

**Discussion by fakuivan** [termux/termux-app#2877] [Q&A] [answered]
Is there any way to access storage providers from termux itself? The wiki has some info on accessing $HOME from other apps, but not the other way around.

**Comment by agnostic-apollo** [comment, marked as answer]
Check github action builds for `termux-api` app and package. Not released officially yet.

https://github.com/termux/termux-api-package/commit/eaf966bf4671f5d97926791cfc08e93cecfe6f31

https://github.com/termux/termux-api/pull/476
